modules/oadp-configuring-aws-bsl.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-configuring-aws-bsl.adoc_{context}"]
+= Configuring the backup storage location using AWS
+
+You can configure the {aws-short} backup storage location (BSL) as shown in the following example.
+
+.Prerequisites
+
+* You have created an object storage bucket using {aws-short}.
+* You have installed the {oadp-short} Operator.
+
+.Procedure
+
+. Configure the BSL custom resource (CR) as shown in the following example:
++
+.Example backup storage location
+[source,yaml]
```

Comment:
We do not config the BSL resource directly
instead we configure the BSL in the DataProtetionApplication rescoues
@shdeshpa07

Reply:
Thanks Amos. This BSL is a description of all the fields that can be configured. You are right, we generally configure the BSL in the DPA. There are already examples of configuring the BSL in the DPA in our doc set. I have linked those examples as additional resources. Does this sound good to you Thanks.